lib/resty/kafka/broker.lua

```diff
@@ -35,6 +35,14 @@ function _M.send_receive(self, request)
         return nil, err, true
     end
 
+    if self.config.ssl then
+      local _, err = sock:sslhandshake(true, self.host, self.config.ssl_verify)
```

Comment:
style: we use four spaces for ident.

Reply:
the first argument `true` looks wrong to me, it should be `nil` or an userdata `session` or `false`.
I think we can use `false` for now, and add a TODO comment: we'd better support session reuse for better performance.

Reply:
Done. I agree makes sense, but seems Kong team just passes true here:
 https://github.com/Kong/kong/blob/master/kong/plugins/http-log/handler.lua#L78
Which calls this underlying? https://github.com/ledgetech/lua-resty-http/blob/master/lib/resty/http.lua#L174 , so would true not get passed as the first argument? Maybe it is just not doing anything when set to true haha.

Reply:
yeah, I have checked the source code, `true` equals `nil` here.